job.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: e2e-test-ae25b65a36f98f9a2bb6cba0affb8cd6f2927980
+  namespace: devnet-issue-567
+spec:
+  ttlSecondsAfterFinished: 120
+  template:
+    metadata:
+      labels:
+        pokt.network/purpose: e2e-tests
+    spec:
+      initContainers:
+        - name: check-services
+          image: ruby:3.0
+          command: ["/bin/sh"]
```

Comment:
This script feels like a copy-paste of the other one.

Again:
1. Need comments on why we need this and what's happening
2. Can we put it in a source file?

Reply:
This was not supposed to be committed, my bad.